earth_enterprise/tutorial/GoogleEarthFusionTutorial.spec

```diff
@@ -1,49 +0,0 @@
-%define _prefix /opt/google/share/tutorials/fusion
```

Comment:
Chris let's leave .spec, SConstruct and Makefile in repo? So that the only tutorial data files have been moved.
Then developers will have a chance to run scons install for tutorial, and they will know what to do in case of getting error.